### PR TITLE
media-gfx/gimp: (0/3) fix icon cache update, FHS/Gentoo policy paths. Fix musl build

### PR DESCRIPTION
**Pull request by band-a-prend**
Closes: https://bugs.gentoo.org/905586
Bug: https://bugs.gentoo.org/900148

### CODE REVIEW

File: media-gfx/gimp/gimp-2.99.14-r3.ebuild [hunk 1]
```diff
@@ -134,7 +134,9 @@ src_prepare() {
 	sed -i -e 's/@PYTHON_PATH@/'${EPYTHON}'/' plug-ins/python/pygimp.interp.in || die
 
 	# Set proper intallation path of documentation logo
-	sed -i -e "s/'gimp-@0@'.format(gimp_app_version)/'gimp-@0@.@1@'.format(gimp_app_version, gimp_app_version_micro)/" data/images/meson.build || die
+	sed -i -e "s/'gimp-@0@'.format(gimp_app_version)/'gimp-${PVR}'/" data/images/meson.build || die
+
+	sed -i -e 's/#ifdef __gnu_linux__/#if defined (__gnu_linux__) \&\& defined (__GLIBC__)/' app/core/gimpbacktrace-backend.h || die #900148
```
Comment (thesamesam) [inline media-gfx/gimp/gimp-2.99.14-r3.ebuild:139]: Please patch this instead, it's too easy for this to become stale.

Comment (band-a-prend) [reply media-gfx/gimp/gimp-2.99.14-r3.ebuild:139]: Should I also patch `data/images/meson.build` to provide version? It's depend on revision number.

Comment (band-a-prend) [reply media-gfx/gimp/gimp-2.99.14-r3.ebuild]: I now use prepared patch from https://github.com/gentoo/gentoo/pull/31087 - now this pull request depends on it.

Comment (thesamesam) [reply media-gfx/gimp/gimp-2.99.14-r3.ebuild]: It's ok to sed meson.build for PVR because it changes based on some information from the ebuild.

**Comment by thesamesam** [comment]
Can you rebase this please?

**Comment by band-a-prend** [comment]
Thanks!

**Pull request closed without merging**